scss/utilities/_variables.scss

```diff
@@ -38,3 +35,11 @@ $transition: 0.14s ease-in-out;
         opacity: 0.8;
     }
 }
+
+
+/* Consultmate variables */
+$mobile-view: max-width: 600px;
+$tab-view: max-width: 1024px;
+$desk-view: min-width 1200px;
+// border thickness
+$b-thickness: 1px;
```

Comment:
Possibly confusing kebab? Rename variable to $border-width to keep in line with W3 terminology?